capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -63,20 +61,19 @@ export class WikiMediaComponent implements OnChanges, OnInit {
     let isOldRequest = false;
     this.wikiService.search(queryString, 'en', 'History')
       .subscribe((result: WikiServiceResult) => {
-        this.error = '';
-        if (result.history && currentQuery === this.cityName) {
+        if (result.history) {
+          this.error = '';
           this.body = result.history;
           this.history = result.history;
           this.urls = result.furtherReading;
           this.title = result.title;
           this.loading = false;
           this.langlinks = result.langlinks;
-          this.prevWordForHistory = 'History';
-          this.prevPreFix = 'en';
+          this.UrlForAttribution = 'https://en.wikipedia.org/wiki/' + result.originalSearchQuery;
         }
-        else if (!isOldRequest) {
-          this.error = 'API did not return a valid response.';
-          console.error('API did not return a valid response.');
+        else {
+          this.error = 'Unfortunately there is no history to display.';
```

Comment:
The error state should be reflected in a test.

Reply:
Hmm, ok, I'll work on adding a test for that.

Reply:
But I should also log it right?

Reply:
@mbehrlich It's actually not an error, that's just the var name.

Reply:
Well... I still think there should be a test of the "empty" state if possible.